Makefile

```diff
@@ -842,6 +848,9 @@ else
 ifeq ($(findstring s390x,$(UNAME_M)),s390x)
 DESTCPU ?= s390x
 else
+ifeq ($(findstring OS/390,$(shell uname -s)),OS/390)
+DESTCPU ?= s390x
+else
```

Comment:
Curious, why this is being added between s390x and s390?

Reply:
I don't think there is a particular reason, @gabylb can you confirm? I can move it below s390 if that is more preferable

Reply:
Yeah, I'd think that's more logical.

Reply:
Probably to group the two `DESTCPU ?= s390x` cases.